### DISCUSSION TITLE
Options for a static site CMS?

### DISCUSSION THREAD

**Discussion by futuresoup** [squidfunk/mkdocs-material#5498] [Q&A] [answered]
There are some great open source CMS platforms that work with markdown static sites, like [Decap CMS](https://decapcms.org/) (formerly Netlify CMS) and [TinaCMS](https://decapcms.org/).

I really need a CMS to collaborate with writers who can't figure out GitHub, Git, or raw Markdown workflows.

How could this be implemented?

If it's a big lift - any suggestions on the best possible workflow for non-technical writers to contribute to a Material for MKDocs site?

For example, authoring in Notion and using their API? Some sort of notebook environment? The GitHub GUI?

**Comment by futuresoup** [comment, marked as answer]
We are having great results so far with DecapCMS! :)

**Comment by Semiotikal** [comment]
@futuresoup Sorry to duplicate this in your repo too but figured it made more sense here. Noticed your repo initially worked with decap CMS but then migrated away from mkdocs entirely to docusaurus. Were there issues with integrating decap and mkdocs? I'm interested in using both but cant find many examples or docs of the two together.

**Reply by futuresoup** [reply]
Hi @Semiotikal - Decap CMS will work in both just fine in either Mkdocs or Docusaurus. The setup is the same if I recall.

We migrated to Docusaurus because our project contributors wanted to build React components for UI customization and were limited by the jinja templating. I kind of wish I stuck with Material for MkDocs because I'm not a coder and it's much easier to work with. It also has better defaults.
